rand_isaac/Cargo.toml

```diff
@@ -18,9 +18,15 @@ travis-ci = { repository = "rust-lang-nursery/rand" }
 appveyor = { repository = "alexcrichton/rand" }
 
 [features]
-serde1 = ["serde", "serde_derive"] # enables serde for BlockRng wrapper
+serde1 = ["serde", "serde_derive", "rand_core/serde1"]
+std = []
```

Comment:
I don't follow why you added an extra `std` feature... Or does testing serialization need std?

Reply:
Yes, exactly. The test uses `BufWriter`, `BufReader` and `bincode`, all requiring `std`.

Reply:
What do you think about only enabling `std` when testing, or even only when testing in combination with serde? Than we don't need the otherwise useless feature flag.

Reply:
Seems a bit fragile. What if you want to test Serde without `std`? Anyway, this is not possible AFAIK.